### PR TITLE
Docker v2

### PR DESCRIPTION
**Pull request by FebruaryRain**
### Summary of changes
Docker file
Will install some required packages
Will not close immediately, has a waiting.py app that keeps the container alive (ease of use for others)
Has some docs too.

### Checklist for submitter
 - [ ] Is there anything the reviewer needs to manually test?
 - [ ] Have you checked if any of the documentation needs to change?

### CODE REVIEW

**Review by ashketchupppp — COMMENTED** [review]
See comments

File: docs/Docker_README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Notes on how to use Docker
```
[the remark this answers is not in the document]
Comment (FebruaryRain) [reply docs/Docker_README.md:1]: Agreed, implemented.

File: Docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:latest
+FROM python:latest
+RUN apt-get update -y
+COPY ./sensors/ /opt/sensors
+COPY ./dashboard /opt/dashboard
+COPY ./util /opt/util
+RUN ["pip", "install", "Counterfit"]
```
[the remark this answers is not in the document]
Comment (FebruaryRain) [reply Docker/Dockerfile:7]: While I would usually dislike leaving cruft in a file going to `master`, I will comment out the unneeded section as we are forced to work fast, so I want a fast reversion on this file only. 
Yes, it's the lazy option. 
Agreed, implemented.

File: Docker/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:latest
```
[the remark this answers is not in the document]
Comment (FebruaryRain) [reply Docker/Dockerfile:1]: By convention perhaps but it isn't a technical requirement. 
The correct command is documented and easily used. 
Given the scope of this issue was to just find the basic image you need and spike what else is needed we can skip this for now and address in a later PR.

File: docs/Docker_README.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+# Move to the Git Repo
+$ cd /path/to/software-engineering-2
+```
+Then to start building:
+```
+$ docker build -t test1 -f .\Docker\Dockerfile .
+```
+Run the container:
+```
+$ docker run -it test1
```
[the remark this answers is not in the document]
Comment (FebruaryRain) [reply docs/Docker_README.md:15]: Good point. 
This is already documented 5 lines down:
```
$ docker container create test1
$ 5370e25a407835e225fa68074d318837d491e91f219da638f5eba33d10ee0307    # <container id example>
$ docker container start 5370e25a407835e225fa68074d318837d491e91f219da638f5eba33d10ee0307
```

**Comment by FebruaryRain** [comment]
Comments responded to. 
Please pull into `master` or issue new comments.

File: docs/Docker_README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (billyrturner) [reply docs/Docker_README.md:15]: @FebruaryRain just checking, you're not using Docker Desktop, right? We're no longer allowed to have it installed due to a change in licensing.

Good alternative is Podman https://podman.io/. Podman also has a docker compatible command line, so you can alias podman to docker, so usual docker commands still work 😄 
```echo "alias docker=podman" >> ~/.zshrc```

[the remark this answers is not in the document]
Comment (billyrturner) [reply docs/Docker_README.md:15]: Might be worth adding a note about Docker desktop, but otherwise I am content

**Review by billyrturner — APPROVED** [review]
Approved provided people are aware of the fact that Docker Desktop licensing has changed, and we should no longer be using it on IBM machines

**Pull request merged**